larpandoracontent/LArThreeDReco/LArPfoMopUp/RecursivePfoMopUpAlgorithm.h

```diff
@@ -0,0 +1,87 @@
+/**
+ *  @file   larpandoracontent/LArTwoDReco/LArClusterMopUp/RecursivePfoMopUpAlgorithm.h
+ *
+ *  @brief  Header file for the recursive pfo mop up algorithm that runs other algs:
+ *          Recusively loop over a series of algorithms until no more changes are made
+ *
+ *  $Log: $
+ */
+#ifndef LAR_RECURSIVE_PFO_MOP_UP_ALGORITHM_H
+#define LAR_RECURSIVE_PFO_MOP_UP_ALGORITHM_H 1
+
+#include "Pandora/Algorithm.h"
+
+namespace lar_content
+{
+
+/**
+ *  @brief  RecursivePfoMopUpAlgorithm class
+ */
+class RecursivePfoMopUpAlgorithm : public pandora::Algorithm
+{
+public:
+    RecursivePfoMopUpAlgorithm() : m_maxIterations(10){};
```

Comment:
Add doxygen `@brief default constructor` etc. and Pandora style is to separate implementation from prototype. Can either inline below, or for a simple algorithm constructor, add implementation near top of .cc file.